notebooks/simple-decoding-example.ipynb

```diff
@@ -108,9 +108,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "subcom_data_decoded = s1isp.decoder.SubCommutatedDataDecoder().decode(subcom_data)\n",
-    "subcom_data_decoded_dict = decoded_subcomm_to_dict(subcom_decoded=subcom_data_decoded)\n",
-    "subcom_data_decoded_df = pd.DataFrame(subcom_data_decoded_dict)"
+    "subcom_data_records_decoded = s1isp.decoder.SubCommutatedDataDecoder().decode(subcom_data)\n",
```

Comment:
Added print and changed name

Reply:
sorry for not being clear, the point was that the `subcom_data` passed in input to the `SubCommutatedDataDecoder().decode` method does not exists.